frontend/packages/dev-console/integration-tests/support/commands/app.ts

```diff
@@ -25,7 +28,26 @@ after(() => {
 });
 
 afterEach(() => {
-  // checkErrors();
+  checkErrors();
+});
+
+Cypress.Commands.add('pageTitleShouldContain', (title: string) => {
+  cy.get('[data-test-id ="resource-title"]')
+    .should('be.visible')
+    .and('contain.text', title);
+});
+
+Cypress.Commands.add('alertTitleShouldContain', (alertTitle: string) => {
+  cy.byLegacyTestID('modal-title').should('contain.text', alertTitle);
+});
+
+Cypress.Commands.add('clickNavLink', (path: [string, string]) => {
+  cy.get(`[data-component="pf-nav-expandable"]`) // this assumes all top level menu items are expandable
+    .contains(path[0])
+    .click(); // open top, expandable menu
+  cy.get('#page-sidebar')
+    .contains(path[1])
+    .click();
```

Comment:
Same comment as above

Reply:
while pageTitleShouldContain and alertTitleShouldContain are not present, hence retained. However, I removed implementation for clickNavLink which was present in integration-tests-cypress folder